android/src/main/java/com/segment/analytics/kotlin/android/Storage.kt

```diff
@@ -92,6 +92,10 @@ class AndroidStorage(
     override fun removeFile(filePath: String): Boolean {
         return eventsFile.remove(filePath)
     }
+
+    override suspend fun rollover() {
```

Comment:
can we add some docs around what this function should be doing

Reply:
good call. will do